tests/unit/helpers/sectorHistories.test.js

```diff
@@ -30,6 +30,26 @@ describe('updateHistoryOnSectorUpdate', () => {
     createHistoryStub.restore();
   });
 
+  it('should create sectot history if no previous one', async () => {
+    SectorHistoryMock.expects('findOne')
+      .withExactArgs({ auxiliary: auxiliaryId, $or: [{ endDate: { $exists: false } }, { endDate: null }] })
+      .chain('lean')
+      .once()
+      .returns(null);
+
+    const result = await SectorHistoryHelper.updateHistoryOnSectorUpdate(auxiliaryId, sector.toHexString(), companyId);
+
+    expect(result).not.toBeDefined();
```

Comment:
Est-ce qu'il y a une difference entre `not.toBeDefined()` et `toBeUndefined()` ? On utilise les deux dans les tests.

Reply:
j'ai l'impression que la diff est assez meta mais j'ai aussi lu que `toBeUndefined` n'etait pas juste `! toBeDefined` donc dans le doute autant utilisé `toBeUndefined` tu as raison